src/components/DayPickerRangeController.jsx

```diff
@@ -321,10 +335,10 @@ export default class DayPickerRangeController extends React.PureComponent {
         && !prevFocusedInput
         && didFocusChange
       )
+      || !isDateVisible
     ) {
       const newMonthState = this.getStateForNewMonth(nextProps);
-      const { currentMonth } = newMonthState;
-      ({ visibleDays } = newMonthState);
+      ({ currentMonth, visibleDays } = newMonthState);
```

Comment:
Not real feedback: Why does this need to be wrapped in parens like this? 🤔

Reply:
@noratarano I thiiiink it’s to destructure into a `let` assignment—this is also the first time I’ve seen this :)

Reply:
yup you are right!